eduvpn/actions/activate.py

```diff
@@ -17,12 +17,17 @@
 from eduvpn.exceptions import EduvpnAuthException, EduvpnException
 from eduvpn.crypto import common_name_from_cert
 from eduvpn.brand import get_brand
+from eduvpn.metadata import Metadata
+gi.require_version('Gtk', '3.0')
```

Comment:
why this was added?

Reply:
Copied from the .pyi file otherwise the type comments use types that are not imported.

Reply:
ok, but the `gi.require_version()` is not related to the typing